app/src/main/java/com/napzak/market/data/mypage/di/MyPageServiceModule.kt

```diff
@@ -0,0 +1,20 @@
+package com.napzak.market.data.mypage.di
+
+import com.napzak.market.core.network.di.JWT
+import com.napzak.market.data.mypage.service.MyPageService
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import retrofit2.Retrofit
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object MyPageServiceModule {
+    @Provides
+    @Singleton
+    fun provideMyPageService(
+        @JWT retrofit: Retrofit,
+    ): MyPageService = retrofit.create(MyPageService::class.java)
```

Comment:
p2: 다음과 같이 바꿔도 정상동작한답니다!
```suggestion
    ): MyPageService = retrofit.create()
```